guides/common/modules/proc_enabling-the-project-client-name-repository.adoc

```diff
@@ -1,7 +1,7 @@
 [id="enabling-the-project-client-name-repository_{context}"]
 = Enabling the {project-client-name} repository
 
-Enable the {Team} {project-client-name} repository for every major version of {RHEL} that you intend to run on your hosts.
+Enable the {project-client-name} repository for every major version of {RHEL} that you intend to run on your hosts.
```

Comment:
Should we generalize this while we're at it? Shouldn't make a difference for Satellite, but makes things clearer for upstream.
```suggestion
Enable the {project-client-name} repository for every major version of {EL} that you intend to run on your hosts.
```